lib/telegraf.py

```diff
@@ -103,8 +104,13 @@ def _write_http_output_config(http_config):
     username = http_config.get('username')
     password = http_config.get('password')
     if username is not None:
-        http_output['username'] = username
-        http_output['password'] = password
+        # Workaround for https://github.com/influxdata/telegraf/issues/4544
+        # http_output['username'] = username
+        # http_output['password'] = password
+        credentials = base64.b64encode(('%s:%s' % (username, password)).encode()).decode('ascii')
```

Comment:
Instead of using string interpolation, use .format.

So, in this case:

```
credentials = base64.b64encode("{}:{}".format(username, password).encode()).decode('ascii')
```